Dockerfile-dev

```diff
@@ -9,4 +9,4 @@ RUN pip install --upgrade pip
 
 COPY . /usr/src/app/
 
-RUN pip install --no-cache-dir -e .[dev]
+RUN pip install --no-cache-dir --process-dependency-links -e .[dev]
```

Comment:
Before merging into master:

1. Push latest BDB-common to pip
2. Remove this label
3. Update version in setup.py